### ISSUE TITLE
What is the testing scope of the dataset in the short video QA?

### ISSUE THREAD

**Issue by yichufan** [rese1f/MovieChat#85]
Thanks for your work! I recently tried to reproduce your work on the short video question-answering task, but the performance always differs from the papaer. I wonder if it might be due to the scope of the dataset used?
For the MSVD-QA dataset, the QA json downloaded from the instructions under `eval_code` includes three subsets: train, test, and val. May I ask if the results in your paper were tested on all of these subsets?
If possible, I would also like to know if the hyperparameter settings are consistent with the code in the repository?

**Comment by yichufan** [comment]
> The hyperparameter settings please refer to the paper. Can you provide the GPT version you use?

Thank you for your response. I tested the GPT-3.5-turbo-0125 API on the entire MSVD dataset (including 50,505 questions), and observed a 10% drop in accuracy and a 0.3 decrease in score.

**Comment by Espere-1119-Song** [comment]
We evaluate MovieChat using the test set and report the average performance across multiple turns of evaluation.
